Pages/_Layout.cshtml

```diff
@@ -28,6 +28,6 @@
 </div>
 
 <script src="_framework/blazor.server.js"></script>
-    <script src="scripts/ClientUtil.js"></script>
+@* <script src="scripts/ClientUtil.js"></script> *@
```

Comment:
I commented this out because the browser kept complaining (in the dev tools) that this file is not found. It seems this change didn't break anything.

Reply:
I deleted the line.